src/Illuminate/Routing/Exceptions/RoutingException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Illuminate\Routing\Exceptions;
+
+use RuntimeException;
+use Illuminate\Routing\Route;
+
+class RoutingException extends RuntimeException
+{
+    /**
+     * The Route object related to the caused error.
+     *
+     * @var \Illuminate\Routing\Route
+     */
+    protected $route;
+
+    /**
+     * Set the Route related to the error.
+     *
+     * @param  \Illuminate\Routing\Route $route
+     * @return $this
+     */
+    public function setRoute(Route $route)
```

Comment:
You're setting it but then nobody has access. IMO, this stuff should be removed, and you should just set this as the exception message in the first place.